web/src/beta/components/PopoverMenuContent/index.tsx

```diff
@@ -0,0 +1,89 @@
+import Icon, { Icons } from "@reearth/beta/components/Icon";
+import { css, styled } from "@reearth/services/theme";
+
+export type MenuItem = {
+  name: string;
+  isSelected?: boolean;
+  icon?: Icons;
+  onClick?: () => void;
+};
+
+export type Props = {
+  width?: string;
+  size: "sm" | "md";
+  items: MenuItem[];
+};
+
+const stylesBySize = {
+  sm: {
+    iconSize: 14,
+    row: css`
+      height: 28px;
+      font-size: 12px;
+      padding: 4px 12px;
+    `,
+  },
+  md: {
+    iconSize: 16,
+    row: css`
+      height: 38px;
+      font-size: 14px;
+      padding: 8px 12px;
+    `,
+  },
+};
+
+const PopoverMenuContent: React.FC<Props> = ({ size, width, items }) => {
+  return (
+    <SRoot width={width}>
+      {items.map((item, i) => {
+        return (
+          <SRow key={i} isSelected={!!item.isSelected} size={size} onClick={item.onClick}>
+            {item.icon && (
+              <SLeftIcon>
+                <Icon icon={item.icon} size={stylesBySize[size].iconSize} />
+              </SLeftIcon>
+            )}
+
+            <SText>{item.name}</SText>
+          </SRow>
+        );
+      })}
+    </SRoot>
+  );
+};
+export default PopoverMenuContent;
+
+const SRoot = styled.div<Pick<Props, "width">>`
+  ${({ width }) => width && `width: ${width};`}
+
+  border-radius: 2px;
+  border: 1px solid #525252;
```

Comment:
Is it possible to use theme colors instead of plain color

Reply:
We plan to update the theme object with new design system variables.
So I'm thinking using current theme variables will make it more difficult to replace.